### ISSUE TITLE
[Features request] a few options for the ''On screen controller''

### ISSUE THREAD

**Issue by zxzzz8** [iina/iina#2267]
im using:
macOS HighSierra 10.13.1 (17B1003)
IINA 1.0.1 Build 103

a few feature ideas for the "On screen controller" ui,
that i think will be really nice to have = 3

1. 
option to make it transparent, and how much transparent it is....

2. 
on the "Floating" layout, make it resizable,
or actually better give an option in the settings to enable it to be resizable....

3. 
give an option in the settings to have a different "Layout" in fullscreen,

4. 
on the "Floating" layout, give an option in the settings to have a different "last position" to fullscreen
[and if you implement number 2, that includes size too]

5. 
for the "auto hide after....", have an option to have a different timer for the mouse pointer

notes

about number 1:
i guess it does already have a very weak transparency? tho that include blur i think?
but thats probably something you had on the library you are using?
so do you think it will be a big issue to make a transparency bar in the settings?

about number 2:
for example, i sometimes want to make it way longer, 
so i can have more accuracy w the mouse-over on the timeline bar, 
especially when i deal w ~8 hour long videos [twitch streams i downloaded to be exact].

about number 3:
honestly that will mostly solve my issue i just mentioned ^here w my long videos, 
by setting it on "floating" in fullscreen and "bottom" in windowed, 
so when i want a more specific control for timeline i just go out of fullscreen....
its also probably much more easy to implement,
but i still think every feature here add in itself.

about number 4:
for example, 
have it as a tiny square in the top left corner when in window mode,
and at the bottom, thin and long, from the middle to the right, in fullscreen mode.

about number 5:

i would personally prefer the mouse pointer to stay a bit longer after the ui is gone....

what im thinking on is a [v] box for making a separate "hide the mouse after...."
that show the option to set an independent time for the mouse to be hidden.

alternatively you can just give the option to set "mouse stay x seconds after the ui",
but that will limit the option to make the mouse disappear before the ui if someone want that,
unless you can give a -x input? XD

sorry if ive done something wrong, im super new to using github X3
i have more things in mind as well, 
but sense they are not related to the "on screen controller"
i won't mention them now, ill probably mention them later in separate feature-issues?

btw should i have separated all or some of these into different feature-issues despite being somewhat related?
[like the mouse pointer hide?], i just thought that would be too spammy....

hope these ideas help, = D
and thank you very much for the awesome app. ^^

**Comment by zxzzz8** [comment]
oh i guess my ideas include a more specific solution to this one? https://github.com/iina/iina/issues/907

so you can merge that other issue i guess?
= X

**Comment by saagarjha** [comment]
Yes, you're correct; the translucency is provided by the system and is not something we can change easily.

**Comment by zxzzz8** [comment]
btw im guessing the ''video image freezes for a second while getting to or from full screen'' 
is also something at an OS level that cannot be easily fix for the same reasons, right?
